extensions/data-workspace/package.json

```diff
@@ -114,13 +118,22 @@
         },
         {
           "command": "projects.openExisting"
+        },
+        {
+          "command": "projects.manageProject",
+          "when": "false"
         }
       ],
       "view/item/context": [
         {
           "command": "projects.removeProject",
           "when": "view == dataworkspace.views.main && viewItem == databaseProject.itemType.project",
           "group": "9_dbProjectsLast@9"
+        },
+        {
+          "command": "projects.manageProject",
+          "when": "view == dataworkspace.views.main",
+          "group": "0_projectsFirst@1"
```

Comment:
Can you put this one above the other command so they're ordered as they'd appear in the UI (/by `group`)?